terraform/example/oidc_connector.tf

```diff
@@ -0,0 +1,19 @@
+# Teleport OIDC connector
```

Comment:
Why commented out?

Reply:
For testing purposes, uncommented it back